dataland-backend/src/test/kotlin/org/dataland/datalandbackend/service/DataManagerTest.kt

```diff
@@ -1,27 +1,35 @@
 package org.dataland.datalandbackend.service
 
-import org.dataland.datalandbackend.model.DataIdentifier
-import org.dataland.datalandbackend.model.StorableDataSet
+import org.dataland.datalandbackend.edcClient.api.DefaultApi
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
 import org.springframework.boot.test.context.SpringBootTest
 
 @SpringBootTest
-class InMemoryDataStoreTest {
+class DataManagerTest {
+
+    val testManager = DataManager(edcClient = DefaultApi(basePath = "dummy"))
 
-    val testStore = InMemoryDataStore()
     val testCompanyNamesToStore = listOf("Imaginary-Company_I", "Fantasy-Company_II", "Dream-Company_III")
 
+    /*
     val testDataSetsToStore = listOf(
         StorableDataSet(companyId = "1", dataType = "SomeDataType", data = "some_data_in_specific_structure_yyy"),
         StorableDataSet(companyId = "1", dataType = "AnotherDataType", data = "some_data_in_specific_structure_iii"),
         StorableDataSet(companyId = "1", dataType = "RandomDataType", data = "some_data_in_specific_structure_aaa")
     )
+    */
```

Comment:
warum ist hier so viel auskommentierter code? das sollte weg sein, bevor der PR in den Review geht!

Reply:
Hamid und ich haben uns entschieden, das einfach rauszunehmen. Gründe:

1.) Die Sachen werden in den e2e-Tests sowieso getestet und sind somit gecovert.

2.) Vorher waren diese Unit-Tests einfach umzusetzen, weil die Daten in den InMemoryDataStore geschrieben wurden. Jetzt werden Daten aber auf den edc-Dummyserver geschrieben.
D.h. beim Speichern findet eine Kommunikation zwischen Backend und einem Server statt.
Das über Mocks hinzubekommen, ist bestimmt irgendwie möglich (auch wenn wir zwei nicht wissen wie).
**Aber**: Warum sich die Mühe machen, wenn es keinen Mehrwert bringt (siehe Punkt 1)

Reply:
ok

Reply:
gelöscht.  fixed

Reply:
ok